### ISSUE TITLE
Add docs on eclipse debug/run process

### ISSUE THREAD

**Issue by RX14** [NOVA-Team/NOVA-Gradle#7]

**Comment by RX14** [comment]
What about this for docs:

Eclipse users should either:
- Use IDEA
- Commit seppuku

**Comment by calclavia** [comment]
@RX14 Or do a PR to NOVA Gradle to add Eclipse support...

**Comment by RX14** [comment]
@calclavia It's impossible, eclipse doesn't have the required features to debug processes started by gradle.

**Comment by skyem123** [comment]
Get @Victorious3 to make it work! :stuck_out_tongue:

**Comment by RX14** [comment]
I doubt @Victorious3 of all people would want to learn the gradle tooling API to make an eclipse plugin.

**Comment by skyem123** [comment]
Well... @Victorious3 wants to develop with Eclipse, so they have to make it work.

**Comment by RX14** [comment]
No, Skye, Vic will simply have to run `gradlew --debug-jvm run17Client` and use the attach an external debugger runconfig.

**Comment by skyem123** [comment]
Then Vic should make the tutorial?

**Comment by RX14** [comment]
Yeah, that's what this issue was about originally, but I don't use eclipse.

**Comment by Victorious3** [comment]
Why would I launch _anything_ with gradle?

**Comment by RX14** [comment]
@Victorious3 because it's the easiest and best way, and it clearly works.

**Comment by Victorious3** [comment]
Neither of which.

**Comment by RX14** [comment]
@Victorious3 I don't see @calclavia complaining that debugging his mods doesn't work, and I **KNOW** that this method is the easiest and least-buggy method to run minecraft, instead of generating gradlestart bullshit that would require the minecraft sources to be on the path. I'm not generating n eclipse projects for n wrappers.
